app/views/hiring_staff/vacancies/summary.html.haml

```diff
@@ -1,21 +1,25 @@
-- content_for :page_title_prefix, "Your job posting for #{@school.name} has been submitted"
+- content_for :page_title_prefix, "The job listing for #{@school.name} has been completed"
 
 =render partial: 'school_vacancy_breadcrumb'
 
 .vacancy.grid-row
-  .column-full
+  .column-two-thirds
     .govuk-box-highlight.margin-bottom-large
-      %h1.heading-large.mv0
+      %h1.heading-large.mt0
         = t('jobs.submitted')
 
+      - if @vacancy.publish_today?
+        %p= link_to t('jobs.preview_posted_job'), job_path(@vacancy), class: 'reversed-link'
+      - else
+        %p= link_to t('jobs.preview_posted_job'), school_job_path(@vacancy.id), class: 'reversed-link'
+
+      %p.mb0 You can also view, amend or delete it on #{link_to "your school's dashboard", new_sessions_path, class: 'reversed-link'}
+
   .column-two-thirds
-    - if @vacancy.publish_today?
-      %p
-        = t('jobs.view_posted_job')
-      %p
-        = link_to job_url(@vacancy), job_path(@vacancy)
-    - else
-      %p
-        = t('jobs.preview_posted_job', date: @vacancy.publish_on)
-      %p
-        = link_to school_job_url(@vacancy.id), school_job_path(@vacancy.id)
+    %h3.heading-medium What happens next
```

Comment:
Can this be extracted to the locale file please along with the rest of the text in this page.